Mathlib/FieldTheory/Galois/Infinite.lean

```diff
@@ -180,9 +180,7 @@ lemma fixingSubgroup_fixedField (H : ClosedSubgroup (K ≃ₐ[k] K)) [IsGalois k
 /-- The Galois correspondence from intermediate fields to closed subgroups. -/
 def IntermediateFieldEquivClosedSubgroup [IsGalois k K] :
     IntermediateField k K ≃o (ClosedSubgroup (K ≃ₐ[k] K))ᵒᵈ where
-  toFun := fun L =>
-    { L.fixingSubgroup with
-      isClosed' := fixingSubgroup_isClosed L }
+  toFun := fun L => ⟨L.fixingSubgroup, fixingSubgroup_isClosed L⟩
```

Comment:
Can you do this? (untested)
```suggestion
toFun L := ⟨L.fixingSubgroup, fixingSubgroup_isClosed L⟩

Reply:
It works (the latest version for finite case also wrote like this)